Lib/_strptime.py

```diff
@@ -547,15 +546,14 @@ def _strptime(data_string, format="%a %b %d %H:%M:%S %Y"):
                 del err
                 bad_directive = bad_directive.replace('\\s', '')
                 if not bad_directive:
-                    raise ValueError("stray %% in format '%s'" % format) from None
+                    raise ValueError(f"stray %% in format {format!r}") from None
```

Comment:
```suggestion
                    raise ValueError(f"stray % in format {format!r}") from None
```
It was doubled for %-formatting.